src/main/java/Bank/State/Interest.java

```diff
@@ -28,4 +29,40 @@ public void setBalance(double balance) {
     public void calculateInterest() {
         interestState.calc();
```

Comment:
Tutaj jest `interestState.calc();` natomiast Ty tworzysz nową funkcję `public double calculate()` to nie ma prawa działać.

W takim razie zamiast  `interestState.calc();`  powinno być  `interestState.calculate();` , wtedy funkcje `calc()` można usunąć